internal/uvm/scsi/manager.go

```diff
@@ -152,14 +155,19 @@ func (m *Manager) AddVirtualDisk(
 	}
 	var mcInternal *mountConfig
 	if mc != nil {
+		var dmverity *guestresource.DeviceVerityInfo
+		if !mc.GuestReadVerity {
+			dmverity = readVerityInfo(ctx, hostPath)
+		}
```

Comment:
Should we get rid of this in favor of always reading the verity info in the guest?

Reply:
this makes sure that newer shims can work with older GCS.

Reply:
Do we have that guarantee with our Confidential support right now? We've already broken older GCS/shim pairings with the scsi refactoring we did.